### ISSUE TITLE
unable to export to local file system

### ISSUE THREAD

**Issue by tomchiverton** [haiwen/seadroid#362]
Pick a file, choose export, no option to save.

Sony Z1, Android 4.4, latest Seadroid from Google app store.

**Comment by Logan676** [comment]
the export feature is used to export the file selected to other Apps, if you want to save the file to your local file system, please use download feature.

**Comment by tomchiverton** [comment]
What download feature? Tapping the item menu doesn't list one, and long pressing the item doesn't do anything. A brief tap offers some open with options, but even for .txt files, download isn't one of them. 
-- 
Tom
Sent from my phone.

**Comment by Logan676** [comment]
> A brief tap offers some open with options, but even for .txt files, download isn't one of them.

It will open with options after downloaded the file, that means the file already saved on your local file system.

There are two ways to download a file.
* Simply tap the file your want to download, it will open a downloading page just like what you have done.
* Click the download menu as shown below.
![download](https://cloud.githubusercontent.com/assets/2975850/7978619/ec8281a4-0ac9-11e5-9d6e-41675c035927.png)

But if you want to save the file into a custom folder, it was not supported yet. You can use a third party App to do that.

**Comment by tomchiverton** [comment]
Ah ha. There's no indication you can scroll the drop down sideways for other options. That's a different bug though. 

As you say, there is no support yet for download to a local folder. That's why this bug exists. 
-- 
Tom
Sent from my phone.

**Comment by shoeper** [comment]
Files are automatically saved at /sdcard/Seafile/AccountName/LibraryName/FolderName/FileName.FileType (or something very similar to that). You should be able to copy them from there to any folder you like and you can also open them from there with any app you like. 
Not optimal because one can not choose between local and external storage so far but it is okay.

**Comment by tomchiverton** [comment]
Yes, I understand that is a workaround e.g. find the hidden download option in Seadroid, then find the hidden folder in Fx. That's not the point. This bug is about there being no easy 'save it here' option in Seadroid itself, directly.

**Comment by shoeper** [comment]
> Yes, I understand that is a workaround e.g. find the hidden download option in Seadroid, then find the hidden folder in Fx. That's not the point. 

I don't agree with this.

> This bug is about there being no easy 'save it here' option in Seadroid itself, directly.
I wouldn't say this is a bug. It's more a feature request and it is already possible to achieve the requested behaviour.

I agree with your point, that it could be a little bit more intuitive.

**Comment by Logan676** [comment]
sorry we won\`t add support to export files to the local storage in the following several version, you can move files by a third party App.

**Comment by tomchiverton** [comment]
Oh, you've closed the feature request. So this will never happen then ?

**Comment by shoeper** [comment]
> sorry we won`t add support to export files to the local storage in the following several version, you can move files by a third party App.

> [..] find the hidden download option in Seadroid, then find the hidden folder in [..]
there is nothing hidden.Neither the download nor the Seafile directory are hidden in any way.

**Comment by tomchiverton** [comment]
Like I said. Different issue, see #366 

Can you reopen this feature request for supporting the correct 'save to file system' intention ?

**Comment by Logan676** [comment]
This feature request is pretty much like the one described in #272, and I have already implemented it in #319. Closing the issue doesn`t mean it will never be handled.

Again, Google have denied the write access to external SD card Since Kitkat (Android 4.4 API 19), so customizing downloading directory doesn\`t work on Kitkat unless the device was rooted. Now a majority of our users are using Kitkat devices, which has placed this feature in a lower priority according to the dev plan.
